packages/cli-platform-android/src/commands/buildAndroid/index.ts

```diff
@@ -0,0 +1,140 @@
+import {
+  CLIError,
+  getDefaultUserTerminal,
+  isPackagerRunning,
+  logger,
+} from '@react-native-community/cli-tools';
+import {Config} from '@react-native-community/cli-types';
+import execa from 'execa';
+import {getAndroidProject} from '../../config/getAndroidProject';
+import {getTaskNames, toPascalCase} from '../runAndroid/runOnAllDevices';
+import adb from '../runAndroid/adb';
+import getAdbPath from '../runAndroid/getAdbPath';
+import {startServerInNewWindow} from '../runAndroid';
+
+export interface BuildFlags {
+  mode: 'debug' | 'release';
+  variant?: string;
+  activeArchOnly?: boolean;
+  packager: boolean;
+  port: number;
+  terminal: string;
+  tasks?: Array<string>;
+}
+
+export async function checkPackager(args: BuildFlags, config: Config) {
+  if (!args.packager) {
+    return Promise.resolve();
```

Comment:
it's already a promise because we're in async fn
```suggestion
    return;
```